docs/getting_started/quickstart.rst

```diff
@@ -0,0 +1,165 @@
+
+Faster ``concat`` across multiple dataframes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our previous ``read_csv`` example operated on a relatively small dataframe. In the following example, we duplicate the same taxi dataset 100 times and then concatenate them together, resulting in a dataset around 19GB in size.
+
+.. code-block:: python
+
+  start = time.time()
+
+  big_pandas_df = pandas.concat([pandas_df for _ in range(25)])
+
+  end = time.time()
+  pandas_duration = end - start
+  print("Time to concat with pandas: {} seconds".format(round(pandas_duration, 3)))
+
+.. code-block:: python
+
+  start = time.time()
+
+  big_modin_df = pd.concat([modin_df for _ in range(25)])
+
+  end = time.time()
+  modin_duration = end - start
+  print("Time to concat with Modin: {} seconds".format(round(modin_duration, 3)))
+
+  print("Modin is {}x faster than pandas at `concat`!".format(round(pandas_duration / modin_duration, 2)))
+
+Modin speeds up the ``concat`` operation by more than 60X, taking less than a second to create the large dataframe, while pandas took close to a minute.
+
+
+Faster ``apply`` over a single column
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The performance benefits of Modin becomes aparent when we operate on large gigabyte-scale datasets. For example, let's say that we want to round up the number across a single column via the ``apply`` operation. 
+
+.. code-block:: python
+
+  start = time.time()
+  rounded_trip_distance_pandas = big_pandas_df["trip_distance"].apply(round)
+
+  end = time.time()
+  pandas_duration = end - start
+  print("Time to apply with pandas: {} seconds".format(round(pandas_duration, 3)))
+
+.. code-block:: python
+  
+  start = time.time()
+
+  rounded_trip_distance_modin = big_modin_df["trip_distance"].apply(round)
+
+  end = time.time()
+  modin_duration = end - start
+  print("Time to apply with Modin: {} seconds".format(round(modin_duration, 3)))
+
+  print("Modin is {}x faster than pandas at `apply` on one column!".format(round(pandas_duration / modin_duration, 2)))
+
+Modin is more than 30X faster at applying a single column of data, operating on 130+ million rows in a second.
+
+Summary
+--------
+
+Hopefully, this tutorial demonstrated how Modin delivers significant speedup on pandas operations without the need for any extra effort. Throughout example, we moved from working with 100MBs of data to 20GBs of data all without having to change anything or manually optimize our code to achieve the level of scalable performance that Modin provides.
+
+Note that in this quickstart example, we've only shown ``read_csv``, ``concat``, ``apply``, but these are not the only pandas operations that Modin optimizes for. In fact, Modin covers `more than 90\% of the pandas API <https://github.com/modin-project/modin/blob/master/README.md#pandas-api-coverage>`, yielding considerable speedups for many common operations.
```

Comment:
Might be useful to add this part of troubleshooting here too:
"If you would like to request a particular method be implemented, feel free to open an
issue. Before you open an issue please make sure that someone else has not already
requested that functionality."

Reply:
I would argue against adding this in the quickstart, given that we already have this in the FAQ section. The quickstart guide is aimed at showing the best of what Modin can do. If the user finds something is not implemented, they will find the FAQ page with the open request blurb.